src/modules/Security.rb

```diff
@@ -688,6 +708,11 @@ def Write
       true
     end
 
+    SYSCTL_VALUES = {
```

Comment:
Better solution than the original, no doubt about it. But I'm not sure if we have some policy about constants defined in the middle of the file. I would have expected it at the top.